app/views/organisations/cohorts/show.html.erb

```diff
@@ -14,49 +14,60 @@
-              <% @presenter.milestone_completion_status.each do |target, data| %>
-                <div>
-                  <div class="flex justify-between">
-                    <p class="text-xs font-medium"><%= "M#{target.milestone_number}: #{target.title}" %></p>
-                    <div class="flex space-x-2">
-                      <p class="text-xs font-medium text-gray-500"><%= "#{data[:students_count]}/#{@presenter.total_students_count}" %></p>
-                      <p class="text-center text-xs font-medium rounded-sm px-1 bg-gray-200"><%= "#{data[:percentage]}%" %></p>
+        <div class="p-4 bg-gray-50 rounded-lg mt-8">
+          <div class="max-w-5xl mx-auto">
+            <h3 class="text-xs font-semibold"><%= 'Student Distribution by Milestone Completion' %></h3>
+            <div class="mt-4">
+              <div class="grid gap-x-8 gap-y-6">
+                <% @presenter.milestone_completion_status.each do |target, data| %>
+                  <div>
+                    <div class="flex justify-between">
+                      <p class="text-xs font-medium"><%= "M#{target.milestone_number}: #{target.title}" %></p>
+                      <div class="flex space-x-2">
+                        <p class="text-xs font-medium text-gray-500"><%= "#{data[:students_count]}/#{@presenter.total_students_count}" %></p>
+                        <p class="text-center text-xs font-medium rounded-sm px-1 bg-gray-200"><%= "#{data[:percentage]}%" %></p>
+                      </div>
                     </div>
+                    <%= link_to students_organisation_cohort_path(milestone_completed: "#{target.id};M#{target.milestone_number}: #{target.title}") do %>
+                      <div class="relative bg-gray-200 rounded-md overflow-hidden mt-0.5">
+                        <div style="<%= "width: #{data[:percentage]}%" %>" class="h-2 text-center bg-green-500"></div>
+                      </div>
+                    <% end %>
                   </div>
-                  <%= link_to students_organisation_cohort_path(milestone: "#{target.id};M#{target.milestone_number}: #{target.title}") do %>
-                    <div class="relative bg-gray-200 rounded-md overflow-hidden mt-0.5">
-                      <div style="<%= "width: #{data[:percentage]}%" %>" class="h-2 text-center bg-green-500"></div>
-                    </div>
-                  <% end %>
-                </div>
-              <% end %>
+                <% end %>
+              </div>
             </div>
           </div>
         </div>
       </div>
+    </section>
+  <% else %>
+    <div class="max-w-5xl mx-auto text-center justify-center items-center">
+      <p class="text-center text-sm italic mt-4 pt-4" ><%= 'There are no students in this cohort.' if total_students_count == 0 %></p>
+      <p class="text-center text-sm italic mt-4 pt-4" ><%= 'There are no milestone targets.' if @presenter.milestone_targets.empty? %></p>
     </div>
-  </section>
+  <% end%>
```

Comment:
Looks like your formating is not working as expected

Reply:
I was using the same HTML beautifier formatter 

It looks like it is not formatting when left a space on the left and no space on the right (vice versa) like that but for other cases it is formatting.

Is it working in that case for you?

But manually added an extra space to the right.